superbench/runner/ansible.py

```diff
@@ -34,7 +34,8 @@ def __init__(self, config):
             if inventory_file or inventory_list:
                 self._config['host_pattern'] = 'all'
                 inventory = InventoryManager(loader=DataLoader(), sources=inventory_file or f'{inventory_list},')
-                host_list = inventory.get_groups_dict()['all']
+                host_list = inventory.get_hosts(pattern='all', order='sorted')
+                self._first_host = host_list[0].get_name()
                 if len(host_list) > 0:
```

Comment:
rename to head_host?

Reply:
what if the length is 0?

Reply:
updated, it will raise exception in update_mpi_config